core/controllers/editor_test.py

```diff
@@ -2937,3 +3016,127 @@ def test_rights_handler_returns_appropriate_rights(self):
         self.assertFalse(response['can_manage_voice_artist'])
 
         self.logout()
+
+
+class ImageUploadHandler(BaseEditorControllerTests):
+    """Test the handling of uploads by image uploads handler."""
+
+    def test_return_error_when_image_not_uploaded(self):
+        """Test error when no image is uploaded."""
```

Comment:
I think it's fine to include the docstring, but I don't think we require them for test functions. If you do keep the docstring, make sure it's a complete sentence here and elsewhere

Reply:
I removed it since I don't think it is required. I found out that many classes do not have a docstring when the class name itself justifies the purpose.

Reply:
I removed the docstring since it doesn't seem to be required